### PR TITLE
Support subdomains

### PR DESCRIPTION
**Pull request by possibilities**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,7 +31,9 @@ function handleDgram(chunk, rinfo) {
         hostname.fqdn.push(chunk.slice(++offset, targetIndex).toString());
     }
 
-    hostname.machineName = hostname.fqdn[0];
+    var tld = hostname.fqdn[hostname.fqdn.length - 2];
+
+    hostname.machineName = tld;
```
Comment (joechrz) [inline index.js]: If I'm reading this right it'll match whatever comes before the actual TLD (`.docker` by default).  So `tld` doesn't seem like the right variable name here.  Could you rename this something like `machineName` or something like that?

Happy to merge this one in too with that minor change.

**Comment by jontg** [comment]
@possibilities ping — I'd love to see this merged in (and we at SalesforceIQ love contributors!); do you mind responding to the PR feedback?  I know it took us almost 2 months to notice... (whoops!)

**Comment by possibilities** [comment]
Updated, thanks!

**Comment by jontg** [comment]
LGTM!

**Comment by jontg** [comment]
(I'll wait for @joechrz to cut a release since he's got push access on npmjs.org) Thank you @possibilities  for the contribution!

**Pull request merged**